BruTile/FileSystem/FileTileSource.cs

```diff
@@ -8,16 +8,16 @@
 
 namespace BruTile.FileSystem;
 
-public class FileTileProvider : ITileProvider
+public class FileTileSource
```

Comment:
Deleted the ITileProvider (which was lie a tile source but with less options). So there are only 'Sources' now. FileTileSource, HttpTileSource, MbTilesTileSource. The WMSC and TMS logic uses the HttpTileSource now (instead of their own Source or Provider implementation).